packages/semver2/semver2.1.2.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "info@dividat.com"
+authors: ["Dividat AG"]
+synopsis: "Semantic version handling for OCaml"
+license: "MIT"
+homepage: "https://github.com/dividat/ocaml-semver"
+bug-reports: "https://github.com/dividat/ocaml-semver/issues"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {>= "1.2.0"}
+  "angstrom" {>= "0.14.0"}
+  "ounit" {with-test & >= "1.0.2"}
+  "yojson" {with-test & >= "1.4.1"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
```

Comment:
The url stanza pointing at the sources is missing

Reply:
Try `opam-publish https://github.com/dividat/ocaml-semver/archive/refs/tags/1.2.0.tar.gz` it should take care of that.